app/src/main/res/values/component_colors.xml

```diff
@@ -205,4 +205,13 @@
   <color name="component_color_mark_topics_completed_background_color">@color/color_palette_mark_chapters_background_color</color>
   <color name="component_color_mark_topics_completed_text_mark_completed_color">@color/color_palette_mark_completed_text_color</color>
   <color name="component_color_mark_topics_completed_text_all_color">@color/color_palette_mark_completed_all_text_color</color>
-</resources>
+  <!-- Audio Language Activity -->
+  <color name="component_color_audio_language_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_language_items_background_color">@color/color_palette_container_background_color</color>
+  <color name="component_color_language_items_heading_text_color">@color/color_palette_dark_text_color</color>
+  <!-- Reading Text Size Activity -->
+  <color name="component_color_reading_text_size_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_reading_text_size_items_heading_text_color">@color/color_palette_dark_text_color</color>
+  <!-- App Language Activity -->
+  <color name="component_color_app_language_activity_background_color">@color/color_palette_dark_background_color</color>
+</resources>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
okay sure, thanks you.

Reply:
@rt4914 can you please check it now I added EOF.